services/community-site/src/pages/RunNode/Table/TestnetValidatorRow.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
```

Comment:
I don't think this is enough. We may not hit the testnet indexer for these nodes and I'm sure we don't have a `testnetDpos` contract instance that we can call `isEligible` on